examples/hybrid_canister/test/tests.ts

```diff
@@ -0,0 +1,155 @@
+import * as dns from 'node:dns';
+dns.setDefaultResultOrder('ipv4first');
+
+import { getCanisterId, Test } from 'azle/test';
+import { createActor as createActorServer } from './dfx_generated/server';
+import { createActor as createActorServerInitAndPostUpgrade } from './dfx_generated/server_init_and_post_upgrade';
+import { createActor as createActorCanister } from './dfx_generated/canister';
+
+export function getTests(): Test[] {
+    const canisterIdServer = getCanisterId('server');
```

Comment:
These should all be broken up into their own files canisterTests.ts serverTests.ts and serverInitAndPostUpgradeTests.ts

Reply:
The file is very short and there is just one method to test, I think that would add unnecessary complexity to the tests

Reply:
Ignore the previous comment, this has been addressed